tests/bluetooth/tester/overlay-le-audio.conf

```
@@ -134,3 +134,7 @@ CONFIG_MCTL_REMOTE_PLAYER_CONTROL_OBJECTS=y
 CONFIG_MCTL_REMOTE_PLAYER_CONTROL=y
 CONFIG_BT_MPL=y
 CONFIG_UTF8=y
+
+# TMAP
+CONFIG_BT_TMAP=y
+CONFIG_BT_DEBUG_TMAP=y
```

This is a legacy debug config. Please use
```suggestion
CONFIG_BT_TMAP_LOG_LEVEL_DBG=y
```